logdna/logdna.py

```diff
@@ -111,7 +112,9 @@ def send_request(self):
                    'mac': self.mac if self.mac else None,
                    'tags': self.tags if self.tags else None},
                stream=True,
-               timeout=self.request_timeout)
+               timeout=self.request_timeout,
+               headers={'User-Agent': self.user_agent}
```

Comment:
we are using `user-agent` instead of `User-Agent` as described [here](https://github.com/logdna/nodejs/blob/master/lib/logger.js#L144)

Reply:
ok.